apps/frontend/src/pages/Account.tsx

```diff
@@ -1,5 +1,102 @@
+import {
+  Box,
+  Divider,
+  Drawer,
+  List,
+  ListItem,
+  ListItemButton,
+  ListItemText,
+  Typography,
+} from "@mui/material";
+import { useState } from "react";
+import YourProfile from "../components/Account/YourProfile";
+import YourListings from "../components/Account/YourListings";
+import YourReviews from "../components/Account/YourReviews";
+import { useStyles } from "../styles/pageStyles";
+
 const Account = () => {
-  return <h1>Account</h1>;
+  const [selectedComponent, setSelectedComponent] = useState<JSX.Element>(
```

Comment:
I would recommend handling this with 3 separate pages that share a layout/sidebar. That way, the user can just navigate to /myreviews or whatever the link would be, and they can also use the forward/back arrows to navigate between the pages

Reply:
Sounds good! Done :)

Profile navigates to `/user`
My Reviews navigates to `/user/reviews`
My Listings navigates to `/user/listings`

Reply:
cc: @ChloeZacharias - I'm thinking the hamburger in the navbar for mobile will have the option for users to navigate to My Profile, My Reviews, and My Listings. Upon clicking these options, the user will navigate to the links as described above. 

I'm making sure that the My Profile, My Reviews, and My Listings pages will be compatible with both mobile and desktop views.

Let me know if you have questions or something doesn't make sense :)